cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,80 @@
+                        <label for="course-id">${_('Course ID')}</label>
+                        <input id="course-id" type="text" name="course-id" aria-required="true"
+                               value="${form_data['course_id']}"
+                               placeholder="${_('e.g. edX+DemoX+Demo_Course')}"/>
+                        % if error:
+                        <div class="error">${msg}</div>
+                        % endif
+                    </li>
+                    <li class="field-radio">
+                        <div>
+                            <label for="dry-run" class="label" data-tooltip="${_('Trial or practice run the task')}">
+                            ${_('Dry Run')}
+                            </label>
+                            % if form_data['is_dry_run']:
+                            <% is_checked = 'checked' %>
+                            % else:
+                            <% is_checked = '' %>
+                            % endif
+                            <input type="checkbox" id="dry-run" name="dry-run" ${is_checked}/>
+                        </div>
+                    </li>
+                </ol>
+            </fieldset>
+        </div>
+        <div class="actions">
+            <button type="submit" class="action action-primary" aria-disabled="true">${_('Force Publish Course')}
+            </button>
+            <a href="${view['url']}" class="action action-secondary action-cancel"
+               data-tooltip="${_('Reset values')}">${_('Reset')}</a>
+        </div>
+    </form>
+    % if current_versions:
+    <div class="result">
+        % if form_data['is_dry_run']:
+        <article>
+            <div class="head-output">
+                ${_('You have done a dry run of force publishing the course. Nothing has changed. Had you run it, the following course versions would have been change.')}
+            </div>
+            <div class="main-output">
+                ${_('The published branch version, {published}, was reset to the draft branch version, {draft}.').format(
+                    published=current_versions['published-branch'],
+                    draft=current_versions['draft-branch']
+                )}
+            </div>
+        </article>
+        % elif updated_versions:
+        <article>
+            <div class="head-output">
+            ${_('Forced published the course {course_id}.').format(
+                course_id=form_data['course_id']
+            )}
+            </div>
+            <div class="main-output">
+            ${_('The published branch version, {published}, was reset to the draft branch version, {draft}.').format(
+                published=current_versions['published-branch'],
+                draft=current_versions['draft-branch']
+            )}
+            </div>
+        </article>
+        % endif
+    </div>
+    % elif not error:
+    <div class="wrapper-error">
+        <div class="error" role="alert">${msg}</div>
```

Comment:
this div should be visible on the page at page load (otherwise the role=alert will not work).  Put the conditional inside this element.